crates/rome_js_parser/src/lib.rs

```diff
@@ -354,68 +354,67 @@ pub struct ParseDiagnostic {
 
 #[derive(Debug, Default, Clone)]
 struct ParserAdvice {
-    advice: Vec<(String, Option<TextRange>, FileId)>,
-    note: Option<String>,
+    /// A list code frame details. The tuple contains the following:
```

Comment:
```suggestion
    /// A frame details list. The tuple contains the following:
```

Not sure about this change but I don't understand what *A list code frame details* is. What's the *list* part referring to? Is it a list of code frames?